### PR TITLE
CI: improvements for gpexpand and gppkg

### PR DESCRIPTION
**Pull request by jchampio**
Two cleanups for the pipeline template:
- fix the inclusion of `gpexpand` in the Concourse pipeline groups when only `centos6` and `CLI` are being flown
- move the `concourse/scripts/gppkg_behave.sh` implementation into the (only) behave test that actually requires it

For the latter, [here's a successful pipeline run](https://dev.ci.gpdb.pivotal.io/teams/main/pipelines/gpdb-dev-pchampion/jobs/gppkg/builds/2). See commit messages for details.

### CODE REVIEW

File: gpMgmt/test/behave/mgmt_utils/steps/mgmt_utils.py [hunk 1]
```diff
@@ -1948,6 +1948,47 @@ def impl(context, gppkg_name):
...
+        # Now copy all the files over.
+        hosts = set(get_all_hostnames_as_list(context, 'template1'))
+
+        host_opts = []
+        for host in hosts:
+            host_opts.extend(['-h', host])
+
+        subprocess.check_call([
+            'gpscp',
+            '-rv',
```
Comment (kalensk) [inline gpMgmt/test/behave/mgmt_utils/steps/mgmt_utils.py]: what is `-r`?

Comment (jchampio) [reply gpMgmt/test/behave/mgmt_utils/steps/mgmt_utils.py]: `-r` is recursive, yeah. I don't know whether all options are passed through; I kind of doubt it.

Comment (jmcatamney) [reply gpMgmt/test/behave/mgmt_utils/steps/mgmt_utils.py]: Nope, it doesn't pass anything through that's not [in the docs](https://gpdb.docs.pivotal.io/590/utility_guide/admin_utilities/gpscp.html).  It wasn't intended as a full wrapper for `scp`, just a way to do a basic copy to a bunch of different hosts.

Comment (kalensk) [reply gpMgmt/test/behave/mgmt_utils/steps/mgmt_utils.py]: Yeah I was skimming the `--help` and online docs and didn't easily see anything about `-r` or recursive. And reading between the lines about it supporting `-r`.

**Comment by jchampio** [comment]
Rebased to latest to keep this PR from rotting; I'm waiting for the `gppkg` job to be fixed before merging.

**Comment by dkrieger-pvtl** [comment]
> I'm waiting for the `gppkg` job to be fixed before merging.

That team said on Friday morning that they will look into it, so I'd expect that to happen soon.

**Pull request merged**